src/pages/[lang]/index.astro

```diff
@@ -53,10 +53,10 @@ updateLang(Astro.url.pathname);
     <Cta
       title={t("home.cta.title")}
       subtitle={t("home.cta.subtitle")}
-      action={{
+      action={undefined /* {
```

Comment:
I would then completely remove this property. If you explicitly left this for the text below, it's fine, but I guess we can also restore it later from git history.